charts/kyverno-policies/templates/baseline/disallow-capabilities.yaml

```diff
@@ -0,0 +1,58 @@
+{{- $name := "disallow-capabilities" }}
+{{- if eq (include "kyverno-policies.podSecurityBaseline" (merge (dict "name" $name) .)) "true" }}
+{{- include "kyverno-policies.supportedKyvernoCheck" (dict "top" . "ver" ">= v1.6.0-rc1") }}
```

Comment:
Do we need to consider the prerelease version as well , or we can just use v1.6.0 here ?

Reply:
That depends entirely on what git tags are exposed. For tests to pass before v1.6.0, must also allow pre-releases otherwise tests won't pass until v1.6.0 tag exists..\

Reply:
I re-read docs more carefully, I updated to be `1.6.0-0` which should catch pre-releases.